File: 49-group-anagrams/49-group-anagrams.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        hashT = {}
        for s in strs:
            sorted_str = ''.join(sorted(s))
            if sorted_str in hashT.keys():
                hashT[sorted_str].append(s)
            else:
                hashT[sorted_str] = [s]
        return hashT.values()

File: 49-group-anagrams/49-group-anagrams.js
/**
 * @param {string[]} strs
 * @return {string[][]}
 */
var groupAnagrams = function(strs) {
    const hashT = {};
    for (let str of strs){
        const sorted = str.split('').sort().join('');
        if (hashT[sorted]) hashT[sorted].push(str);
        else hashT[sorted] = [str];
    }
    return Object.values(hashT)
};